jsonnet/components/telemeter-client.libsonnet

```diff
@@ -59,20 +85,79 @@ function(params) {
                     volumeMounts+: [
                       {
                         mountPath: '/etc/kube-rbac-policy',
-                        name: 'telemeter-client-kube-rbac-proxy-config',
+                        name: 'secret-' + $.kubeRbacProxySecret.metadata.name,
                         readOnly: true,
                       },
                     ],
                   }
                 else
                   c,
               super.containers,
-            ),
+            ) +
+            [
+              {
+                name: 'kube-rbac-proxy-metric',
```

Comment:
this is adding a new kube-rbac-proxy container while we should be overriding the existing one.